packages/jest-runtime/src/index.ts

```diff
@@ -553,6 +553,69 @@ export default class Runtime {
       return globals;
     }
 
+    if (specifier.startsWith('data:')) {
+      if (
+        this._shouldMock(
+          referencingIdentifier,
+          specifier,
+          this._explicitShouldMockModule,
+          {conditions: this.esmConditions},
+        )
+      ) {
+        return this.importMock(referencingIdentifier, specifier, context);
+      }
+
+      const fromCache = this._esmoduleRegistry.get(specifier);
+
+      if (fromCache) {
+        return fromCache;
+      }
+
+      const match = specifier.match(
+        /^data:text\/javascript;(charset=utf-8|base64),(.*)$/,
+      );
+
+      if (!match) {
+        throw new Error('Invalid data URI');
+      }
+
+      let code = match[2];
+      if (match[1] === 'base64') {
+        code = atob(code);
```

Comment:
`atob` and `btoa` are only available on later versions of node I think. You can use `Buffer.from(code, 'base64').toString()`